new_builder_evaluation/app/domain/evaluation.py

```diff
@@ -158,6 +156,40 @@ def downloads_scoring_datasets(
                 final_datasets.append(final_dataset)
         return final_datasets
 
+    def get_no_scoring_datasets(self, task_id: int):
```

Comment:
The method names should be not_scoring, instead of no_scoring

Reply:
done